### ISSUE TITLE
[ENH] Color code progress bars for detectors and motors.

### ISSUE THREAD

**Issue by ambarb** [NSLS-II/bluesky#839]
Can the progress bar for detectors be a different color?

**Comment by danielballan** [comment]
Oh, when you mentioned this I thought you wanted the _variables_ to be different colors, as in customizing the code highlighting in IPython. This would be much simpler, and can be done with an IPython profile without changing the bluesky library code. (It still might make sense to pull into bluesky itself, but you wouldn't _need_ any bluesky changes to try it out.)

**Comment by ambarb** [comment]
Actually, that might also be handy, especially if multiple detector lists are defined.  Maybe it would save some heart ache with `^R` and such.

**Comment by danielballan** [comment]
Yeah. The scope of _that_ [custom code highlighting] is probably a job for a summer student.
